### PR TITLE
Fix links

### PR DESCRIPTION
**Pull request by mdellweg**

### CODE REVIEW

File: staging_docs/sections/help/community/get-involved.md [hunk 1]
```diff
@@ -23,13 +23,12 @@ Introduce yourself, your project, what you’re hoping to achieve with Pulp. We
 
 - For reporting a **bug** or requesting a **feature**:
     1. First try to search if some similar issue was not already reported.
-    2. If not, submit an issue to the appropriate repository (pulpcore, pulp_rpm, pulp-oci-images, etc)
-- For **security issue**, send an email to [pulp-security@redhat.com](#) with:
+    2. If not, submit an issue to the appropriate repository (pulpcore, pulp\_rpm, pulp-oci-images, etc)
+- For **security issue**, send an email to [pulp-security@redhat.com](mailto:pulp-security@redhat.com) with:
```
Comment (pedro-psb) [inline staging_docs/sections/help/community/get-involved.md:27]: `pulp_rpm` is rendering correctly. What is the reason to escape the underscore?

Comment (mdellweg) [reply staging_docs/sections/help/community/get-involved.md:27]: It was marked red in my editor...

Comment (mdellweg) [reply staging_docs/sections/help/community/get-involved.md:27]: Also:
https://stackoverflow.com/questions/35827838/how-to-show-underscores-symbol-in-markdown#35829104

Comment (mdellweg) [reply staging_docs/sections/help/community/get-involved.md:27]: `_` is a syntax element in markdown. Even if it does not complain when it does not come in pairs.

**Review by pedro-psb — APPROVED** [review]
Fair

**Pull request merged**